CMakeLists.txt

```diff
@@ -25,6 +25,8 @@
 # MATERIALS OR THE USE OR OTHER DEALINGS IN THE MATERIALS.
 
 cmake_minimum_required(VERSION 2.8.12)
+cmake_policy(SET CMP0054 NEW)
```

Comment:
Good catch! Somehow the cmake on my Ubuntu 14.04 got linked to CMake 3.5. So no warnings surfaced when testing.